### PR TITLE
Improvement pack

### PR DESCRIPTION
**Pull request by 4miners**
- display qrcode on address page (closes https://github.com/LiskHQ/lisk-explorer/issues/49)
- add currency display switch (closes https://github.com/LiskHQ/lisk-explorer/issues/48)
- add Bittrex support for Market Watcher (closes https://github.com/LiskHQ/lisk-explorer/issues/53)
- add some known addresses

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -21,15 +21,24 @@
             "port" : 6379,
             "password" : ""
         },
+        "exchnageRates": {
```
Comment (Isabello) [inline config.json]: This typo should be corrected.

File: test/config.json [hunk 2]
```diff
@@ -21,15 +21,24 @@
             "port" : 6379,
             "password" : ""
         },
+        "exchnageRates": {
```
Comment (Isabello) [inline test/config.json]: This should be corrected like the other one.

File: utils/exchange-api/index.js [hunk 3]
```diff
@@ -47,27 +79,30 @@ module.exports = function (config) {
...
-
-    if (exchanges.LISKBTC.hasOwnProperty(config.liskbtcExchange)) {
-        config.liskbtcExchange = exchanges.LISKBTC[config.liskbtcExchange];
-        console.log('Exchange:', util.format('Configured %s as LISK/BTC exchange', config.liskbtcExchange[0]));
-    } else {
-        console.log('Exchange:', 'Warning: Unrecognized LISK/BTC exchange!');
-        console.log('Exchange:', 'Defaulting to Poloniex...');
-        config.liskbtcExchange = exchanges.LISKBTC.poloniex;
-    }
+    _.each(config.exchnageRates.exchanges, function (coin1, key1) {
```
Comment (Isabello) [inline utils/exchange-api/index.js]: All of these `exchnageRates` should read `exchangeRates`

File: utils/exchange.js [hunk 4]
```diff
@@ -1,87 +1,30 @@
...
-            }
-        }.bind(this));
-    };
-
-    this.LISKUSD = function (lisk) {
-        if (lisk && this.LISKBTC !== '~' && this.BTCUSD !== '~') {
-            return (parseFloat(lisk) * parseFloat(this.LISKBTC) * parseFloat(this.BTCUSD)).toFixed(3);
-        } else {
-            return 0;
+        if (!config.exchnageRates.enabled) {
```
Comment (Isabello) [inline utils/exchange.js]: Another typo.

File: utils/exchange.js [hunk 5]
```diff
@@ -1,87 +1,30 @@
...
+                this.tickers = result;
+            }
+        }.bind (this));
     };
 
     // Interval
-
-    if (config.enableExchange) {
-        setInterval(this.loadRates, config.updateExchangeInterval);
+    if (config.exchnageRates.enabled) {
```
Comment (Isabello) [inline utils/exchange.js]: Typo

File: utils/exchange-api/index.js [hunk 6]
```diff
@@ -1,10 +1,17 @@
 var request = require('request'),
-    util = require('util');
+    _ = require('underscore'),
+    util = require('util'),
+    async = require('async');
 
 module.exports = function (config) {
+    // No need to init if exchange rates are disabled
+    if (!config.exchnageRates.enabled) {
```
Comment (Isabello) [inline utils/exchange-api/index.js]: typo

File: app.js [hunk 7]
```diff
@@ -21,8 +21,7 @@ app.set('version', '0.3');
 app.set('strict routing', true);
 app.set('lisk address', 'http://' + config.lisk.host + ':' + config.lisk.port);
 app.set('freegeoip address', 'http://' + config.freegeoip.host + ':' + config.freegeoip.port);
-app.set('fixed point', config.fixedPoint);
-app.set('exchange enabled', config.enableExchange);
+app.set('exchange enabled', config.exchnageRates.enabled);
```
Comment (Isabello) [inline app.js]: typo

**Comment by 4miners** [comment]
Thanks, corrected.

**Comment by 4miners** [comment]
There is issue with header update, please don't merge this PR yet.

File: lib/orders/bittrex.js [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+                return cb('Invalid data received');
+            }
+        });
+    };
+
+    this.acceptOrders = function (data) {
+        var asks = [],
+            bids = [];
+
+        for(var i in data.result.buy)
```
Comment (Isabello) [inline lib/orders/bittrex.js]: JShint hissues start here, can be resolved by changing to the below for line 49 to like 54
```
        /* jshint shadow:true */
        for(var i in data.result.buy) {
            bids.push([data.result.buy[i].Rate, data.result.buy[i].Quantity]);
        }

        for(var i in data.result.sell) {
            asks.push([data.result.sell[i].Rate, data.result.sell[i].Quantity]);
        }
        /* jshint shadow:false */
```

Comment (4miners) [inline lib/orders/bittrex.js]: Fixed.

**Pull request merged**